test/e2e/vsphere_workload_api_test.go

```diff
@@ -22,7 +22,7 @@ func TestVSphereMulticlusterWorkloadClusterAPI(t *testing.T) {
 			api.WithWorkerNodeCount(1),
 			api.WithStackedEtcdTopology(),
 		),
-		vsphere.WithBottleRocket123(),
+		vsphere.WithUbuntu123(),
```

Comment:
Changing this to Ubuntu until we update the worker node validation to also handle BR nodes